server/schemas/schemaModel.js

```diff
@@ -1,10 +1,53 @@
 var mongoose = require("mongoose");
-
+var bluebird = require("bluebird");
+var helpers = require("../config/helpers");
 var Schema = mongoose.Schema;
-
 var schemaSchema = new Schema({
```

Comment:
Yeah, been the plan for awhile now. As of yet, no one has suggested a good replacement. If y'all come to a consensus, I'd be more than happy to amend the commit.